.github/workflows/dotnet.yaml

```diff
@@ -8,32 +8,32 @@ jobs:
         os: [ubuntu-latest, windows-latest, macOS-latest]
     name: Dotnet build
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@master
```

Comment:
It's probably dangerous to track master because it could change. Is there a version that would work?

Reply:
This was switched from master to v1 2 days ago as part of 944622dd65abfd1c5475e1e1463f6624c626a5c2. I'm confused why, as wouldn't you want build & integration tests to run off master?

Reply:
That specifies the version of the action; it has no relation to the ref that is checked out.

This PR should be rebased and all build related commits removed.

Reply:
Apologies, I see that it has been rebased.

The action should not track master as it may eventually break the SxS workaround.